packages/gatsby/src/redux/types.ts

```diff
@@ -75,6 +75,7 @@ export interface IGatsbyNode {
 export interface IGatsbyPlugin {
   name: string
   version: string
+  id?: Identifier
```

Comment:
This can be a required field, as it's added at load time.

Reply:
Good to know, will update :+1: